src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7845,7 +7845,19 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_ImplicitIndexerInitializer" xml:space="preserve">
     <value>implicit indexer initializer</value>
   </data>
-  <data name="IDS_RefStructInterfaces" xml:space="preserve">
+  <data name="IDS_FeatureRefStructInterfaces" xml:space="preserve">
     <value>ref struct interfaces</value>
   </data>
+  <data name="ERR_RuntimeDoesNotSupportByRefLikeGenerics" xml:space="preserve">
+    <value>Target runtime doesn't support by-ref-like generics.</value>
+  </data>
+  <data name="ERR_RefStructConstraintAlreadySpecified" xml:space="preserve">
+    <value>The 'ref struct' is already specified.</value>
```

Comment:
>The 'ref struct' [](http://example.com/codeflow?start=11&length=16)

Perhaps just `'ref struct' is already specified` rather than `The `ref struct` ...`.